sdk/cosmosdb/cosmos/src/request/StatusCodes.ts

```diff
@@ -37,7 +37,7 @@ export const StatusCode = {
 /**
  * @ignore
  */
-export type StatusCode = (typeof StatusCode)[keyof typeof StatusCode];
+export type StatusCode = typeof StatusCode[keyof typeof StatusCode];
```

Comment:
I don't understand this code, but were the parens significant?

Reply:
They are not. It's basically a hacky way to build out an enum that is type compatible with literals